sdk/identity/identity-common/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@azure/identity-common",
```

Comment:
If we add this package.json here, we need to also add it to the rush configuration and set up a full build setup for it. There are pros and cons to that approach. At the end of the day I don't think that's what we want, and we need to make sure these files in identity-common are isolated modules without any external dependencies. That's problematic for the shared MSAL test configuration.

I think the easiest solution for the test configuration is probably just to duplicate it, much as it may pain me.